### ISSUE TITLE
The `value-list-comma-*-after` rules refuse to fix a comma that opens the value

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#134]
Found while reviewing the fix for #126. The refusal is older than that fix, which only made it visible: until then `--fix` swallowed the warning, and now the warning stands and nothing ever clears it.

### What minimal example or steps are needed to reproduce the bug?

```css
a { prop: ,0; }
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "plugins": [
    "@stylistic/stylelint-plugin"
  ],
  "rules": {
    "@stylistic/value-list-comma-space-after": "always"
  }
}
```

### How did you run Stylelint?

```shell
stylelint "**/*.css" --config myconfig.json --fix
```

### Which Stylelint-related dependencies are you using?

```json
{
  "@stylistic/stylelint-plugin": "5.3.0",
  "stylelint": "17.14.1"
}
```

### What did you expect to happen?

```css
a { prop: , 0; }
```

### What actually happened?

The problem is reported and never fixed, however many times `--fix` runs:

```
1:11  Expected single space after "," (@stylistic/value-list-comma-space-after)
```

Two rules refuse a comma they could repair:

- `@stylistic/value-list-comma-space-after` under `always` and `always-single-line`;
- `@stylistic/value-list-comma-newline-after` under `always`.

### Do you have a proposal to fix the bug?

Each of them decides what it may touch by

```js
isFixable: (declNode, index) => index > declarationValueIndex(declNode),
```

and the boundary is off by one. For `a { prop: ,0; }` the declaration is `prop: ,0`, `declarationValueIndex` is 6, and the comma sits at 6 — it is the value's first character, not a comma in the property name, and the whitespace after it belongs to the value like any other. Only a comma *before* that index is out of reach.

`index >= declarationValueIndex(declNode)` is enough for these two. Measured on this branch: `a { prop: ,0; }` becomes `a { prop: , 0; }` under `value-list-comma-space-after` and `a { prop: ,\n0; }` under `value-list-comma-newline-after`, both relinting clean; `a { --a: ,0; }` behaves the same; `a { pr,op: 0,0; }` still keeps the comma in its property name untouched and still reports it; all 138 existing cases of the three rules pass.

**The same change must not be made to `@stylistic/value-list-comma-space-before`.** It has the matching gap — `a { prop: ,0; }` under `never` and `never-single-line` is reported and not fixed — but there the whitespace at stake is the one *before* the comma, and for a comma opening the value that whitespace is `decl.raws.between`, not the value. The rule writes through `setDeclarationValue` and cannot reach it. With `>=` it silences its own warning and changes nothing:

```
without --fix   1:11
with --fix      nothing, and the code is untouched
relinting it    1:11
```

which is a fresh instance of #126. Curing that side means teaching the rule to write `decl.raws.between`, which is a different job from moving a boundary and is not covered here.
